src/utils/google-signin-utils.js

```diff
@@ -14,162 +14,230 @@
  * limitations under the License.
  */
 
-/** Sentinel used to tell parent that we are communicating with it. */
-export const SENTINEL = 'google_signin';
-/** Command name for when the parent frame is ready for iframe start. */
-export const PARENT_READY_COMMAND = 'parent_frame_ready';
-/** Command name for when the Google Sign-in iframe is ready. */
-export const INTERMEDIATE_IFRAME_READY_COMMAND = 'intermediate_iframe_ready';
-/** Command name for when the user's metering parameters are returned from the publisher. */
+import {GetEntitlementsParamsExternalDef} from '../../src/api/subscriptions';
```

Comment:
Is there a way to avoid this import? It looks like it only used for type checks. I would like it if this file doesn't have any imports.

Reply:
The closure compiler uses this type defintion import. This import shouldn't add to the filesize of the compiled JS though, since it's just used by the compiler to verify types. Was your concern related to filesize?

Reply:
Yep that was my concern. Thanks for explaining it to me.

Reply:
Np!